pkg/controllers/binding/binding_controller.go

```diff
@@ -61,7 +81,10 @@ func (c *ResourceBindingController) Reconcile(ctx context.Context, req controlle
 
 	isReady := helper.IsBindingReady(binding.Spec.Clusters)
 	if !isReady {
-		klog.Infof("ResourceBinding(%s/%s) is not ready to sync", binding.GetNamespace(), binding.GetName())
+		msg := fmt.Sprintf("ResourceBinding %s is not ready.", req.NamespacedName.String())
+		klog.Infof(msg)
+		c.recorder.Event(binding, corev1.EventTypeWarning, "ResourceBindingNotReady", msg)
+
```

Comment:
```
Events:
  Type     Reason                   Age                 From                Message
  ----     ------                   ----                ----                -------
  Warning  ResourceBindingNotReady  44s                 binding-controller  ResourceBinding default/nginx-deployment is not ready.
```

This basically adds the `warning` event to `all` ResourceBinding objects. Are you sure you want to do this?